draft-ietf-tls-semistatic-dh.md

```diff
@@ -198,17 +195,14 @@ in use.
 
 # Certificate Format
 
-Like signing keys, static DH keys are carried in the Certificate
+Similar to signing keys, static DH keys are carried in the Certificate
 message, either directly in the EE certificate, or in a delegated
 credential. In either case, the OID for the SubjectPublicKeyInfo
 MUST be appropriate for use with (EC)DH key establishment. If
-in a certificate, the key usage and EKU MUST also be set appropriately
-See {{I-D.ietf-curdle-pkix}} and [[TBD: P-256, etc.]] for specific
-details about these formats.
+in a certificate, the key usage and EKU MUST also be set appropriately.
+See {{?I-D.ietf-curdle-pkix}} for specific details about these formats.
```

Comment:
I know you aren't really changing this, but I think that this needs to be a normative reference.

Reply:
Yeah, probably. I'll make that change!